server/app/assets/javascripts/map.ts

```diff
@@ -0,0 +1,48 @@
+import {Map} from 'maplibre-gl'
+
+export const init = () => {
+  const geoJsonDataObject = window.app.data.maps || {}
+  const mapElements = document.querySelectorAll('[id^="cf-map-"]')
+
+  mapElements.forEach((mapElement) => {
```

Comment:
So you loop over each item in `window.app.data.maps` you can key the key (id) and value (geojson). That'll simplify the everything.